scripts/jenkins/Dockerfile

```diff
@@ -43,13 +43,15 @@ RUN export OPENMPI_VERSION=4.0.0 && \
     export OPENMPI_URL=https://www.open-mpi.org/software/ompi/v${OPENMPI_VERSION_SHORT}/downloads/openmpi-${OPENMPI_VERSION}.tar.bz2 && \
     export OPENMPI_ARCHIVE=openmpi-${OPENMPI_VERSION}.tar.bz2 && \
     export OPENMPI_SOURCE_DIR=openmpi-${OPENMPI_VERSION} && \
+    export OPENMPI_INSTALL_PREFIX=/usr && \
     wget --quiet ${OPENMPI_URL} --output-document=${OPENMPI_ARCHIVE} && \
     echo "${OPENMPI_SHA1} ${OPENMPI_ARCHIVE}" | sha1sum -c && \
     mkdir -p ${OPENMPI_SOURCE_DIR} && \
     tar -xf ${OPENMPI_ARCHIVE} -C ${OPENMPI_SOURCE_DIR} --strip-components=1 && \
     cd ${OPENMPI_SOURCE_DIR} && mkdir build && cd build && \
-    ../configure --prefix=/usr --with-cuda && \
+    ../configure --prefix=${OPENMPI_INSTALL_PREFIX} --with-cuda && \
     make -j8 install && \
+    echo "btl_smcuda_use_cuda_ipc = 0" >> ${OPENMPI_INSTALL_PREFIX}/etc/openmpi-mca-params.conf && \
```

Comment:
Interesting - so there was no configure-time option to do this? One always must edit the configuration parameters after the fact?

Reply:
Not that this is an issue - just curious.

Reply:
As far as I know

Reply:
`mpirun --mca btl_smcuda_use_cuda_ipc 0`?

Reply:
we didn't want to override this option on systems where the IPC actually works

Reply:
Isn't that what the `MPIEXEC_PREFLAGS` variable is for?

Reply:
> Isn't that what the `MPIEXEC_PREFLAGS` variable is for?

Good point